Ports/zig/patches/0005-Extend-support-for-SerenityOS-target.patch

```diff
@@ -5,108 +5,11 @@ Subject: [PATCH] Extend support for SerenityOS target
 
 Co-Authored-By: Linus Groh <mail@linusgroh.de>
 ---
- zig/lib/std/c.zig                    |  2 +-
- zig/lib/std/debug/Dwarf/abi.zig      | 17 ++++++-
- zig/lib/std/debug/SelfInfo.zig       |  7 +--
```

Comment:
`std.debug` was reworked upstream, I used the opportunity to upstream my patches after adapting them: https://github.com/ziglang/zig/pull/25457